### DISCUSSION TITLE
Navigation UI light color palette?

### DISCUSSION THREAD

**Discussion by pamelahu** [twilio-labs/paste#3353] [General] [answered]
Hi,

Curious if there are any updates on there being a light color palette version of the new navigation UI?

**Comment by SiTaggart** [comment, marked as answer]
Hi @pamelahu 

We were not able to prioritize that variant of the sidebar due to capacity constraints this quarter. With the release of the Navigation UI kit there is at least an option available to use in the interim. Coupled with the gradient bar those will hopefully fulfill the immediate requirements.

We can potentially look to revisit prioritizing that variant in Q4 when weighed up against other prioritizes.
